src/test/java/ca/team2706/frc/robot/commands/mirrorable/MirrorableTest.java

```diff
@@ -0,0 +1,153 @@
+package ca.team2706.frc.robot.commands.mirrorable;
+
+import edu.wpi.first.wpilibj.command.CommandGroup;
+import edu.wpi.first.wpilibj.command.Subsystem;
+import mockit.Injectable;
+import mockit.Mocked;
+import mockit.Tested;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+public class MirrorableTest {
+
+    @Tested
+    private MirroredCommand a, b, c, d, e;
+
+    @Tested
+    private MirroredCommandGroup f, g, h;
+
+    @Before
+    public void setUp() {
+        a = new MirroredCommand() {
```

Comment:
Since these are all the same you could make a method to return a new object, just to make your life easier.